src/components/elements/query-image.js

```diff
@@ -1,7 +1,6 @@
-// TODO: replace image.js completely with this component
 import React from 'react'
 import styled from 'styled-components'
-import Img from 'gatsby-image'
+import { GatsbyImage, getImage } from 'gatsby-plugin-image'
 import PropTypes from 'prop-types'
 
 export const ImageWrapper = styled.div`
```

Comment:
do we still need to wrap it using imagewrapper?, i think with new one we don't need to right?

Reply:
<img width="440" alt="Screenshot 2021-06-11 at 1 59 43 PM" src="https://user-images.githubusercontent.com/75345074/121638373-71b13280-cabd-11eb-814e-77ed05fe702e.png">

@mustofa-binary for normal situation, yes we do not need as gatsby will auto generate the container for the images and generate the image sets. But for some cases, we still need this for some cases where the images supplied doesn't have proper sizing 🙏🏼